docs/jwt-auth.md

```diff
@@ -0,0 +1,76 @@
+# Bot Authentication for third-party API's
+
+There may be a case where you want your detection bot to make an external call to an API you own. The Forta bot sdk provides the following methods for generating a JWT token which allows bots make authorized requests to external APIs by using the scan node's identity which the detection bot is running on:
+
+- `fetchJwtToken(claims, expiresAt)` See method details [javascript/typescript](sdk.md#fetchjwttoken),[python](python.md#fetchjwttoken)
+- `decodeJwtToken(token)` See method details [javascript/typescript](sdk.md#decodejwttoken),[python](python.md#decodejwttoken)
+
+!!! warning "It is up to the external API to verify the JWT & verify the smart contract in the JWT data"
```

Comment:
It might help to add an example of how an external API can verify the JWT. Also, what is the smart contract in the JWT data?

Reply:
Added an express server example